<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestSpinBox</class>
 <widget class="QWidget" name="TestSpinBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>250</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TestSpinBox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox01Value">
     <property name="title">
      <string>数值输入输出</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_01Count">
            <property name="text">
             <string>数量</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBox_01Count">
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="suffix">
             <string> kg</string>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_01Calculate">
          <property name="text">
           <string>计算</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_02Price">
          <property name="text">
           <string>单价</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_01Price">
          <property name="prefix">
           <string>$ </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_03Total">
          <property name="text">
           <string>总价</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_02Total">
          <property name="prefix">
           <string>$ </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox02Decimal">
     <property name="title">
      <string>进制转换</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_04Dec">
          <property name="text">
           <string>十进制</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_02Dec">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="prefix">
           <string>Decimal </string>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_05Bin">
          <property name="text">
           <string>二进制</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_03Bin">
          <property name="prefix">
           <string>Binary </string>
          </property>
          <property name="displayIntegerBase">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_06Hex">
          <property name="text">
           <string>十六进制</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox_04Hex">
          <property name="prefix">
           <string>Hexadecimal </string>
          </property>
          <property name="displayIntegerBase">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_01Count</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pushButton_01Calculate</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_01Price</sender>
   <signal>valueChanged(double)</signal>
   <receiver>pushButton_01Calculate</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
